src/main/java/seedu/duke/command/ListNoteCommand.java

```diff
@@ -23,22 +23,25 @@
             + PREFIX_DELIMITER + PREFIX_TAG + " TAG1...] "
             + "[up/down]";
 
-    private ArrayList<String> tags;
-    private boolean isSorted;
-    private Boolean isAscendingOrder;
-
+    /**
+     * Gets how the command is expected to be used.
+     *
+     * @return String representation of how the command is to be used.
+     */
     public static String getCommandUsage() {
         return COMMAND_USAGE;
     }
 
+    private ArrayList<String> tags;
+    private Boolean isAscendingOrder;
+
     /**
      * Constructs a ListCommand to list all the Notes in the Notebook in a sorted order.
      *
      * @param isAscendingOrder determines the order of the sorting of the Notes.
      */
     public ListNoteCommand(Boolean isAscendingOrder) {
```

Comment:
Similarly, perhaps we should change to boolean if the parameter is not necessary in defining generics.